mesheryctl/pkg/utils/healthcheck.go

```diff
@@ -161,6 +161,7 @@ func IsMesheryRunning(currPlatform string) (bool, error) {
 			deploymentList, err := deploymentInterface.List(context.TODO(), metav1.ListOptions{})
 
 			if err != nil {
+				err = errors.New("Meshery Server unavailable at http://localhost:13021. Is your Meshery Server up and running?")
```

Comment:
How did you determine that Meshery is up at localhost:13021 ?

It could be dynamic right based on your k8s cluster? Did you determine using your local Meshery instance? Please clarify.

// @lekaf974 @MUzairS15

Reply:
@alphaX86 Port was mentioned in the issue description. I assumed it to be the deafult one.

Reply:
Do we have to make it dynamic?

Reply:
the port can  be different if you run it locally, vs docker vs kubernetes. Perhaps simply ask to validate it is up and running using `meshery system status` command would be better ?

Reply:
This suggestion seems fine to me